sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/RxGatewayStoreModel.java

```diff
@@ -469,7 +470,7 @@ private Mono<RxDocumentServiceResponse> toDocumentServiceResponse(Mono<HttpRespo
                         if (content.refCnt() > 0) {
                             // Unwrap failed before StoreResponse took ownership -> release our retain
                             // there could be a race with the doOnDiscard above - so, use safeRelease
-                            io.netty.util.ReferenceCountUtil.safeRelease(content);
+                            ReferenceCountUtil.safeRelease(content);
```

Comment:
just curious: is this required when we already have doOnDiscard? Suppose any discarded values due to errors should be handled in the doOnDiscard?